### DISCUSSION TITLE
Seperate Account with limited access

### DISCUSSION THREAD

**Discussion by Omegax9** [AzuraCast/AzuraCast#4364] [Q&A]
Hi 

So i am using azuracat on linode and it's very well designed. Thanks developers

Now i want to create separate accounts for each individual users with limited resources. 

For example i want to create an user account with limited amount of HDD space, fix numbers of listeners.. i have noticed that after creating an account user can change listeners amount...Also how do i restrict to one live broadcast per user?

Thanks

**Comment by Omegax9** [comment]
Anyone pls....

**Comment by ivellios1988** [comment]
How to prevent the user from changing the max listeners limit?

Go to Administration -> User accounts -> menu on the left -> Users -> Permissions

First, you need to create a role that will not include permission to edit the station's broadcasting settings. This will prevent the user from changing the max number of listeners. Then you grant this permission set to a selected user.

Setting per-station limits of HDD space:
Edit station -> Administration -> There you will find some links titled 'Manage storage locations and storage quota here'
